src/functions-and-arrays.js

```diff
@@ -29,8 +147,30 @@ const wordsUnique = [
   'bring'
 ];
 
+let uniquifyArray = (arr) => {
+  if (arr.length === 0) {
+    return null;
+  } else {
+  let newArr = []
+  arr.forEach((elem) => {
+    if (!newArr.includes(elem)) {
+      newArr.push(elem);
+    }
+  })
+  return newArr;
+  }
+}
+
 // Iteration #6: Find elements
 const wordsFind = ['machine', 'subset', 'trouble', 'starting', 'matter', 'eating', 'truth', 'disobedience'];
+let doesWordExist = (arr, word) => {
+  if (arr.length === 0) {
+    return null;
+  } else {
+  if (arr.includes(word)) {
```

Comment:
no need to make this else statement cause if the first condition happens the function will stop and return null :)
Example 
```
if(arr.length===0){return null;}
if(arr.includes(word)){return true}
else{return false}
```